src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static DifferentColor getRandomColor() {
+        int index = new Random().nextInt(DifferentColor.values().length);
+        return DifferentColor.values()[index];
```

Comment:
```suggestion
    private final Random random = new Random();
    
    public static DifferentColor getRandomColor() {
        int index = random.nextInt(DifferentColor.values().length);
        return DifferentColor.values()[index];
```
It is better to make Random a class field so that it is not created every time a method is called